### PR TITLE
update grid view for classic farm UI

### PR DESCRIPTION
**Pull request by XiaoYhun**

### CODE REVIEW

File: src/components/YieldPools/FairLaunchPools.tsx [hunk 1]
```diff
@@ -18,22 +22,44 @@ import { useBlockNumber, useModalOpen, useOpenModal } from 'state/application/ho
...
+      <DropdownSVG style={{ rotate: isOpen ? '180deg' : 'none' }} />
+    </ToggleButtonWrapper>
+  )
+}
+
+const FairLaunchPools = ({ fairLaunchAddress, farms, active }: FarmsListProps) => {
   const dispatch = useAppDispatch()
+  const [viewMode] = useViewMode()
+  const above1200 = useMedia('(min-width:1200px)')
+  const above768 = useMedia('(min-width:768px)')
```
Comment (nguyenhoaidanh) [inline src/components/YieldPools/FairLaunchPools.tsx:62]: use constant MEDIA_WIDTH

File: src/components/YieldPools/ListItem.tsx [hunk 2]
```diff
@@ -148,7 +154,9 @@ const ListItem = ({ farm, setSharedPoolAddress }: ListItemProps) => {
   const balance = useTokenBalance(pairAddressChecksum)
   const staked = useStakedBalance(farm.fairLaunchAddress, farm.pid)
   const rewardUSD = useFarmRewardsUSD(farmRewards)
-  const { mixpanelHandler } = useMixpanel()
+
+  const isNotDeposited = !balance.value.gt(0)
```
Comment (viet-nv) [inline src/components/YieldPools/ListItem.tsx:158]: classic farm dont have deposit step, it's only stake. So I suggest that this variable should be `canStake` or isAbleToStake`

File: src/components/YieldPools/ListItem.tsx [hunk 3]
```diff
@@ -171,6 +179,7 @@ const ListItem = ({ farm, setSharedPoolAddress }: ListItemProps) => {
     isStakeInvalidAmount = true
   }
 
+  // This only use to validate in Stake modal
   const isStakeDisabled = isStakeInvalidAmount
```
Comment (viet-nv) [inline src/components/YieldPools/ListItem.tsx:183]: LOL. This is a redundant variable. haha

File: src/components/YieldPools/ListItem.tsx [hunk 4]
```diff
@@ -184,6 +193,7 @@ const ListItem = ({ farm, setSharedPoolAddress }: ListItemProps) => {
     isUnstakeInvalidAmount = true
   }
 
+  // This only use to validate in Unstake modal
   const isUnstakeDisabled = isUnstakeInvalidAmount
```
Comment (viet-nv) [inline src/components/YieldPools/ListItem.tsx:197]: same with isStakeDisabled. 
Remove them will save 2 lines of code 👯‍♂️

File: src/components/YieldPools/ListItem.tsx (hunk 3 above)
Comment (XiaoYhun) [reply src/components/YieldPools/ListItem.tsx:183]: its use to validate if user typein value is valid or not, quite legacy code, but if want to remove need to refactor the modals in this file

Comment (viet-nv) [reply src/components/YieldPools/ListItem.tsx:183]: I know this is a legacy code. I mean at this line: https://github.com/KyberNetwork/kyberswap-interface/pull/1659/files#diff-7a13446d6ec90d398824f960a761ccb4c3877e697337fcdb890df3c3ba5a2ef0R745, we can change it to 
```
disabled={isStakeInvalidAmount}
```

Comment (XiaoYhun) [reply src/components/YieldPools/ListItem.tsx:183]: ah ok :))))

File: src/pages/Pool/index.tsx [hunk 5]
```diff
@@ -184,17 +183,22 @@ export default function PoolCombination() {
...
+  const { search: searchValueInQs = '' } = useParsedQueryString<{
+    search: string
+  }>()
+  const debouncedSearchText = useDebounce(searchValueInQs.trim().toLowerCase(), 300)
 
-  const [searchText, setSearchText] = useState('')
-  const debouncedSearchText = useDebounce(searchText.trim().toLowerCase(), 300)
+  const onSearch = (search: string) => {
+    navigate(location.pathname + '?search=' + search + '&tab=classic', { replace: true })
+  }
```
Comment (viet-nv) [inline src/pages/Pool/index.tsx:201]: I suggest use `useSearchParams` hook

```
const [searchParams, setSearchParams] = useSearchParams()
const searchValueInQs = searchParams.get('search') || ''

  const onSearch = (search: string) => {
     searchParams.set('search', search)
     setSearchParams(searchParams)
  }
```

**Review by viet-nv — APPROVED** [review]

**Pull request merged**